serde_test/src/de.rs

```diff
@@ -496,7 +498,7 @@ impl<'a, I> SeqVisitor for DeserializerSeqVisitor<'a, I>
             Some(&Token::SeqSep) => {
                 self.de.tokens.next();
                 self.len = self.len.map(|len| len - 1);
-                Ok(Some(try!(Deserialize::deserialize(self.de))))
+                Deserialize::deserialize(&mut *self.de).map(Some)
```

Comment:
Yes. Without it:

```
error[E0507]: cannot move out of borrowed content
   --> src/de.rs:501:42
    |
501 |                 Deserialize::deserialize(self.de).map(Some)
    |                                          ^^^^ cannot move out of borrowed content
```

Previously `Deserialize::deserialize` took the argument by mut reference and `self.de` is a mut reference so it was fine. Now `Deserialize::deserialize` takes the argument by value so Rust sees this as moving the reference out of `self`.